api/edit_observation.js

```diff
@@ -0,0 +1,28 @@
+var errors = require('../errors')
+var through = require('through2')
+
+module.exports = function (osm) {
```

Comment:
A comment explaining what this module is for, its inputs & outputs etc, could be helpful for Future Us.